src/mono/wasm/runtime-test.js

```diff
@@ -62,12 +62,13 @@ if (typeof console !== "undefined") {
 		console.error = console.log;
 }
 
-if (typeof crypto == 'undefined') {
+if (typeof crypto === 'undefined') {
 	// /dev/random doesn't work on js shells, so define our own
 	// See library_fs.js:createDefaultDevices ()
 	var crypto = {
 		getRandomValues: function (buffer) {
```

Comment:
This should have a comment that this is a simple insecure polyfill for testing purposes only.

Reply:
Addressed